tests/hypothesis_strategies/kernels.py

```diff
@@ -30,62 +30,62 @@ class KernelType(Enum):
 
 linear_kernels = st.builds(
     LinearKernel,
-    variance_prior=st.one_of(st.none(), priors),
+    variance_prior=st.one_of(st.none(), priors()),
```

Comment:
I guess this was just an error that was overlooked previously? Or why the change?